### PR TITLE
Omit row action menu for deleted rows

### PR DESCRIPTION
**Pull request by mandolyte**
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword/datatable-translatable/66)

### CODE REVIEW

**Comment by mandolyte** [comment]
Testing Notes:
- login
- org unfoldingWord
- resource tN
- language English
- book Jude (as of this writing Jude has deleted rows)
- set rows per page to 100
- scroll down to end of file
- Notice that for deleted rows (no values on target side), the row action menu is absent.

Test row recovery:
- for a deleted row, enter a value for support reference
- click out
- at this point, the row will be recovered and moved to the end of the target file data, above its deleted rows. The action menu will be restored for the row.

**Comment by mandolyte** [comment]
@ancientTexts-net NOTE! this issue did not require any changes to tc-create itself. Only datatable-translatable. Thus yalc should work easily for testing.

File: src/components/actions-menu/ActionsMenu.js [hunk 1]
```diff
@@ -36,6 +36,39 @@ function RowMenu({
...
+  // 8: "May…be multiplied to you\t"
+  // 9: "These ideas are spoken of as if ...
+
+  // Note that there is the constant value "rowHeader" in slot zero.
+  // Let's test by splitting index 1; in the sample above there should
+  // be two elements, both being the string "JUD" if the row is not
+  // deleted.
+  let cellvals = []
+  cellvals = rowData[1].split('\t')
+  if ( cellvals[0] !== cellvals[1] ) {
```
Comment (ancientTexts-net) [inline src/components/actions-menu/ActionsMenu.js:65]: Putting comment in ISSUE with screenshot:

Seems the Actions Menu is also removed from rows that are INSERTED in target, 
but don't exist in the SOURCE.

Is this desired behavior please?

Thanks

**Review by ancientTexts-net — CHANGES_REQUESTED** [review]

**Comment by ancientTexts-net** [comment]
@mandolyte 

Seems the Actions Menu is also removed from rows that are INSERTED in target,
but don't exist in the SOURCE.

Is this desired behavior please?

Thanks

**Comment by mandolyte** [comment]
No! not the desired behavior. Glad you caught that. I had to refine my test to determine if a row is deleted or not. The new test (also in code comments) is this:
```js
  // If a row is deleted (as opposed to being a new add/insert), then
  // source side will *not* be empty, but the target side will be empty.
  let cellvals = []
  cellvals = rowData[1].split('\t')
  // is this a deleted row?
  if ( cellvals[0] !== "" && cellvals[1] === "" ) {
    return (
      <>
      </>
    )
  }
```

**Comment by ancientTexts-net** [comment]
___
*[src/components/actions-menu/ActionsMenu.js, line 65 at r1](https://reviewable.io/reviews/unfoldingword/datatable-translatable/66#-MdwargID5wHs4MAkq63-r1-65:-Mf1IrYU9-PLzL0Lqk5K:bb6ywmx) ([raw file](https://github.com/unfoldingword/datatable-translatable/blob/39e3d5c4281d81977dbfaada60d6b8d437ce96d9/src/components/actions-menu/ActionsMenu.js#L65)):*
<details><summary><i>Previously, ancientTexts-net (Zach P) wrote…</i></summary><blockquote>

Putting comment in ISSUE with screenshot:

Seems the Actions Menu is also removed from rows that are INSERTED in target, 
but don't exist in the SOURCE.

Is this desired behavior please?

Thanks
</blockquote></details>

LGTM

**Review by ancientTexts-net — APPROVED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/unfoldingword/datatable-translatable/66)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @mandolyte)

**Review by ancientTexts-net — COMMENTED** [review]
LGTM

**Pull request merged**